# SLE712_files
These are files which will be used for teaching purposes in SLE712 Bioinformatics and Biotechnology at Deakin University.

## ab1files
These are some Sanger sequencing files used in 2019 but not for 2020.

## bioinfo_asst3_part1_files
This folder contains mydata.tsv, gene_expression.tsv and growth_data.csv. We will work with the mydata.tsv file in class to demonstrate how to properly read a system file into R. The other two files gene_expression.tsv and growth_data.csv will be used by students to answer questions asked in assignment 3 part 1. This part of the assignment is designed to test students ability to read files into R and perform ordering, subsetting, and create simple charts.

## bioinfo_asst3_part2_files
This folder contains example.fa, create_fasta.sh, sample.fa, mutblast_functions and unwrap_fasta.pl. create_fasta.sh is a shell script called that samples 20 random E. coli gene sequences. The sample.fa file is a multi fasta file containing 20 randomly selected E. coli gene sequences. The names of the sequences in sample.fa have been removed because it is part of the exercise for students to figure out to what gene these sequences belong. In class we will work together to analyse a sequence in the example.fa file. The mutblast_functions.R file contains some handy functions that students should use in their assignment to perform random mutagenesis of a sequence and to perform a blast search. The unwrap_fasta.pl file is a perl script to unwrap a fasta file. This part of the assignment is designed to test more advanced skills including using randomisation tools, using BLAST, performing simulations and interpreting data.

